cli/azd/pkg/azure/arm_parameters.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azure
+
+type ArmParameters map[string]ArmParameterValue
+
+type ArmParameterFile struct {
```

Comment:
nit: Should we document similarly to `ArmParameters` for future usage? A link to the schema [docs](https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json) could be nice.

Reply:
Can do!